packages/babel-template/src/index.js

```diff
@@ -1,31 +0,0 @@
-// @flow
```

Comment:
to avoid loosing file edits history, I would suggest having two separate commits (one to rename the files, and other to convert the content)